go.mod

```diff
@@ -154,6 +154,8 @@ require (
 	modernc.org/token v1.1.0 // indirect
 )
 
-replace go.temporal.io/api v1.34.0 => go.temporal.io/api v1.32.1
+// See https://github.com/temporalio/cli/pull/596#discussion_r1666988960
+// #596 downgraded api to 1.32.1, but there are required bug fixes that came after 1.32.1.
+replace go.temporal.io/api v1.34.0 => go.temporal.io/api v1.32.1-cherry-pick-for-cli-tag
```

Comment:
Why are we using a `replace` clause instead of updating the primary `go.temporal.io/api` version above?